src/main/java/org/mastodon/mamut/util/GeometryUtils.java

```diff
@@ -0,0 +1,36 @@
+package org.mastodon.mamut.util;
+
+public class GeometryUtils
+{
+	/**
+	 * Gets the euclidean distances between the two given coordinate arrays. The
+	 * number of dimensions of both coordinate array is expected to be the same.
+	 * Otherwise {@link Double#NaN} is returned
+	 *
+	 * @param coordinates1 the first coordinate array
+	 * @param coordinates2 the second coordinate array
+	 * @return the euclidean distance
+	 */
+	public static double getEuclideanDistance( double[] coordinates1, double[] coordinates2 )
```

Comment:
ImgLib2 has a method `LinAlgHelpers.distance( a, b )` for this.

Reply:
Thanks for pointing this out. Changed to this method.